mozci/task.py

```diff
@@ -358,6 +360,30 @@ def configuration(self):
         parts = config.split("-")
         return "-".join(parts[:-1] if parts[-1].isdigit() else parts)
 
+    @property
+    def overhead(self):
+        """Calculate the overhead of a task.
+
+        The methodology is simple: each task (action) has a start/end time.
+        Each group also has a start/end time. Take the earlist known group start
+        and latest known group end time, ensure the two falls somewhere in between
+        task start/end.
+
+        Then calculate the overhead by taking the difference between the start
+        and end times.
+
+        Returns:
+            float: difference between task start/end and group star/end times.
```

Comment:
```suggestion
            float: difference between task start/end and group start/end times.
```

Reply:
Thanks for spotting them. Sorry about the typos.